Purchases.xcodeproj/project.pbxproj

```diff
@@ -746,6 +742,7 @@
 		2DCB85BF2406EC3F003C1260 /* Recovered References */ = {
 			isa = PBXGroup;
 			children = (
+				37E3564466002C9162AC7C5E /* Package.swift */,
```

Comment:
it got moved to `Recovered References`, does it look fine in the project?

Reply:
Strangely, it looks fine.